### PR TITLE
pass over godoc

### PR DESCRIPTION
**Pull request by thomasten**

### CODE REVIEW

File: attestation/attestation.go [hunk 1]
```diff
@@ -25,15 +25,15 @@ type Report struct {
...
 var ErrEmptyReport = errors.New("empty report")
 
-// VerifyAzureAttestationToken takes a Microsoft Azure Attestation Token in JSON Web Token compact
-// serialization format and verifies the tokens public claims and signature. The Attestation providers
-// keys are loaded from providerURL/certs over TLS and need to be in JSON Web Key format. The validation
-// is based on the trust in this TLS channel. Note, that the token's issuer (iss) has to equal the providerURL,
-// and providerURL must use the HTTPS scheme.
+// VerifyAzureAttestationToken takes a Microsoft Azure Attestation token in JSON Web Token compact
+// serialization format and verifies the token's public claims and signature. The attestation provider's
+// keys are loaded from providerURL over TLS. The validation is based on the trust in this TLS channel.
```
Comment (katexochen) [inline attestation/attestation.go:30]: Shouldn't we mention that the path has to be "/certs"?

File: ecrypto/ecrypto.go [hunk 2]
```diff
@@ -78,6 +78,9 @@ func Decrypt(ciphertext []byte, key []byte) ([]byte, error) {
 }
 
 // SealWithUniqueKey encrypts a given plaintext with a key derived from a measurement of the enclave.
+//
+// Ciphertexts can't be decrypted if the UniqueID of the enclave changes. If you want
+// to be able to decrypt ciphertext across enclave versions, use SealWithProductKey.
 func SealWithUniqueKey(plaintext []byte) ([]byte, error) {
```
Comment (katexochen) [inline ecrypto/ecrypto.go:84]: Maybe you could add a short paragraph to the ecrypto.doc file describing in which scenarios/ for what tasks those functions are used.

File: attestation/attestation.go (hunk 1 above)
Comment (thomasten) [reply attestation/attestation.go:30]: Microsoft is the only party that can operate an attestation server, so I thought that this is an implementation detail the user should not worry about.

Comment (katexochen) [reply attestation/attestation.go:30]: Okay, I'm not so familiar with Azure, I thought you could choose your own path if you create your own provider as mentioned in this line:
[>  or create their own providers with custom policies.](https://docs.microsoft.com/en-us/azure/attestation/basic-concepts#regional-shared-provider)

**Review by katexochen — APPROVED** [review]

**Pull request merged**